ui/src/app/components/workflows/workflows-home/workflows-home.component.html

```diff
@@ -43,9 +43,26 @@ <h3 class="modal-title">Upload workflow</h3>
   </div>
 </clr-modal>
 
-<clr-datagrid (clrDgRefresh)="onClarityDgRefresh($event)">
+<clr-datagrid [(clrDgSelected)]="selected" (clrDgRefresh)="onClarityDgRefresh($event)">
 
   <clr-dg-action-bar>
+    <clr-dropdown>
+      <button type="button" class="btn btn-sm btn-secondary" [disabled]="selected.length <= 0" clrDropdownTrigger>
+        Bulk Actions
+        <clr-icon shape="caret down"></clr-icon>
+      </button>
+      <clr-dropdown-menu clrPosition="bottom-left" *clrIfOpen>
+        <div aria-label="Activate" clrDropdownItem [disabled]="isActivateSelectedWorkflowsDisabled(selected)" (click)="activateSelectedWorkflows(selected)">
+          <clr-icon shape="power"></clr-icon>
+          Activate
+        </div>
+        <div aria-label="Deactivate" clrDropdownItem [disabled]="isDeactivateSelectedWorkflowsDisabled(selected)" (click)="deactivateSelectedWorkflows(selected)">
+          <clr-icon shape="power"></clr-icon>
```

Comment:
I am sorry that I am pointing this out here ... Icons are the same `shape="power"`. I would distinguish it with this class: `<clr-icon shape="power" class="has-badge"></clr-icon>`. 
<img width="199" alt="Screenshot 2020-09-29 at 12 44 36" src="https://user-images.githubusercontent.com/18142017/94548818-90360d00-0251-11eb-96e1-3a22825dc307.png">

+ update single activate/deactivate actions with badge

Reply:
I think you missed this one : update single activate/deactivate actions with badge